plugins/gatsby-source-wagtail/gatsby-node.js

```diff
@@ -22,7 +22,6 @@ exports.sourceNodes = async (
     // not remove the mock data!
     // mock data lives at './test/mock.json'
     if (process.env.USE_MOCK_DATA) {
-      console.warn('Using fake designer data...');
```

Comment:
Is there any harm in leaving this console in? I think its useful to know in the console during build whether we're using fake designer data or not.

Reply:
Opps I did not mean to remove this.  It got caught in my cleanup.